packages/firebase_messaging/README.md

```diff
@@ -110,6 +148,22 @@ By default background messaging is not enabled. To handle messages in the backgr
    <application android:name=".Application" ...>
    ```
 
+#### iOS configuration (Swift)
+1. In the top of `AppDelegate.swift`, add the import of firebase_messaging:
+   
+   ```swift
+    import firebase_messaging
+   ```
+
+1. Then add the following code to `AppDelegate.swift`:
```

Comment:
Maybe you should specify where in `AppDelegate.swift`? E.g. `didFinishLaunchingWithOptions` if that is the case.